macOS ci fails

The macOS CI fails with the following error 
```
Error: No such keg: /usr/local/Cellar/gcc@8
Error: Process completed with exit code 1.
```

https://github.com/dic-iit/bipedal-locomotion-framework/runs/2771521862?check_suite_focus=true#step:6:29

cc @traversaro

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -99,13 +99,6 @@ jobs:
       if: matrix.os == 'macOS-latest'
       run: |
 
-        # https://github.com/actions/virtual-environments/issues/2322
-        rm -f /usr/local/bin/2to3
-
-        # https://github.com/actions/virtual-environments/issues/2391
-        brew unlink gcc@8
-        brew unlink gcc@9
-
         brew update
         brew install ace boost eigen swig qt5 orocos-kdl catch2 qhull ipopt cppad pkg-config pybind11 libmatio spdlog opencv pcl librealsense nlohmann-json
 

